.gitlab-ci-check-apidocs.yml

```
@@ -32,3 +32,54 @@ test:apidocs:
     - swagger-cli validate docs/*.yml
     # Verify that the Swagger docs follow the autodeployment requirements
     - python3 verify_docs.py `find docs -name "*.yml"`
+
+trigger:apidocs:rebuild-mender-api-docs:
+  stage: .post
+  only:
+    changes:
+      - docs/*
+    refs:
+      - /^(master|staging|[0-9]+\.[0-9]+\.x)$/
+  image: alpine
+  before_script:
+    - apk add git python3 py3-pip
+    - pip3 install pyyaml
+    - git clone https://github.com/mendersoftware/integration.git mender-integration
+    - alias release_tool=$(realpath mender-integration/extra/release_tool.py)
+    - apk add --no-cache curl
+  script:
+    # If the repo is not recognized, ignore
+    - if ! echo $(release_tool --list git --all) | grep $CI_PROJECT_NAME; then
+    -  echo "Repository $CI_PROJECT_NAME not found in release_tool. Exiting"
+    -  exit 0
+    - fi
+    # Get the current stable release (latest tag matching the N.N.0 pattern)
+    - CURRENT_STABLE=$(cd mender-integration && git tag | egrep -e '^[0-9]+\.[0-9]+\.0$' | sort -n -r | head -n1 | sed -e 's/\.0$/.x/g')
```

Maybe a bit far fetched, but that sort will chose `2.9.0` in front of `2.10.0`. I think it should work with `-V`
```suggestion
    - CURRENT_STABLE=$(cd mender-integration && git tag | egrep -e '^[0-9]+\.[0-9]+\.0$' | sort -V -r | head -n1 | sed -e 's/\.0$/.x/g')
```

---

Ack (I did not know the `-V` flag!)